internal/msgqueue/v1/rabbitmq/rabbitmq.go

```diff
@@ -231,7 +231,7 @@ func (t *MessageQueueImpl) pubMessage(ctx context.Context, q msgqueue.Queue, msg
 	}
 
 	// if this is a tenant msg, publish to the tenant exchange
-	if !t.disableTenantExchangePubs && msg.TenantID != "" {
+	if !t.disableTenantExchangePubs && msg.TenantID != "" && msg.ID != "task-stream-event" {
```

Comment:
should this be

```
if msg.TenantID != "" && (!t.disableTenantExchangePubs || msg.ID == "task-stream-event")
```

(we want to send stream events even if we've disabled all other pubs?)